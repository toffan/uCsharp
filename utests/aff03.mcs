int main() {
    int a = 5;
    int b = 6;
    b = (a = 2);

    return 0;
}
