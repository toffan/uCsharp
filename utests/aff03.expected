JUMP __main
_main:
    LOADL 5
    LOADL 6
    LOADL 2
    LOAD (1) -1[ST]
    LOADL 3
    STOREI (1)
    LOAD (1) -1[ST]
    LOADL 4
    STOREI (1)
    POP (0) 1

    LOADL 0
    RETURN (1) 0
    RETURN (0) 0
__main:
    CALL (SB) _main
    HALT
